### DISCUSSION TITLE
Reporting facility claims in allowerd amount file

### DISCUSSION THREAD

**Discussion by Rashmi-sb** [CMSgov/price-transparency-guide#546] [Q&A] [answered]
We wanted to evaluate a scenario where a facility claim comes only with revenue code, 
Do we report it like below? and this amount meets the 20 threshold of combination of RC and provider.

  "out_of_network":[{ 
    "name": "XYZ",
    "billing_code_type": "RC",
    "billing_code_type_version": "2020",
    "billing_code": "0121",
    "description": Medical/surgical/GYN",
    "allowed_amounts": [{
      "tin": {
        "type": "ein",
        "value": "1234567890"
      },
      "service_code": ["01", "02", "03"],
      "billing_class": "institutional",
      "payments": [{
        "allowed_amount": 25.00,
        "providers": [{
          "billed_charge": 50.00,
@shaselton-usds  @RLTx1391 @mkress-epic @BobSyracuse

**Comment by Rashmi-sb** [comment]
@shaselton-usds  can you help us here.

**Comment by shaselton-usds** [comment, marked as answer]
Following up on this -- if that is the code that you have for the claim (allowed amount file), then that is what will be reported.
